bin/MadGraph5_aMCatNLO/cards/production/13TeV/exo_diboson/Spin-2/RSGraviton_hh_hVVhbb_fullLep/RSGraviton_hh_hVVhbb_fullLep_narrow_M1000/RSGraviton_hh_hVVhbb_fullLep_narrow_M1000_customizecards.dat

```diff
@@ -0,0 +1,13 @@
+set param_card mass 39 1000
+set param_card decay 39 0.001
+set param_card yukawa 4 0
+set param_card yukawa 5 4.8
+set param_card yukawa 6 172.5
+set param_card mass 4 0
```

Comment:
Is this something which has already been done for other requests?  In principle one should not toggle masses or couplings between 0 and non-zero using the customizecards, since this is not taken into account properly at the diagram generation level.